transfer/batch.go

```diff
@@ -1,7 +1,14 @@
 package transfer
 
+// BatchItemArg is a Git LFS batch item argument.
+type BatchItemArg struct {
+	Key   string
+	Value string
+}
+
 // BatchItem is a Git LFS batch item.
 type BatchItem struct {
 	Pointer
 	Present bool
+	Args    []BatchItemArg
```

Comment:
We can use a map[string]string here instead of defining a new struct

Reply:
I thought about that too, but I used an array of structs so that the ordering could be precisely controlled if need be. That makes it easier to write tests that check the output. WDYT?

Reply:
As an alternative, I could use the `map[string]string` and write the entries in the keys' alphabetical order.